src/pcapi/domain/reimbursement.py

```diff
@@ -129,13 +140,16 @@ class ReimbursementRateForBookAbove20000(payments_models.ReimbursementRule):
     valid_until = None
 
     def is_relevant(self, booking: Booking, cumulative_revenue: Decimal) -> bool:
-        if not booking.stock.offer.type == str(ThingType.LIVRE_EDITION):
+        if non_eligible_type(booking.stock.offer):
+            return False
+        if _is_educational_offer(booking.stock.offer):
```

Comment:
cf commentaire en dessous, ici pour simplifier:
```suggestion
        offer = booking.stock.offer
        if not is_book(offer): # ou directement offer.type != str(ThingType.LIVRE_EDITION)
            return False
        if offer.isEducational: # ou offer.is_educational (mais ici je vois pas trop d'intérêt même future)
```